backend/chainlit/playground/providers/langchain.py

```diff
@@ -65,7 +67,11 @@ async def create_completion(self, request):
 
         messages = self.create_generation(request)
 
-        stream = make_async(self.llm.stream)
+        # https://github.com/langchain-ai/langchain/issues/14980
+        stream = make_async(self.llm.stream)(
+            input=messages,
+            **request.prompt.settings
```

Comment:
This will also fail. There is no longer a `prompt` attribute in the request. See the [anthropic](https://github.com/Chainlit/chainlit/blob/cadb0cb103598d287e9bf91e4cd439c479bee4a6/backend/chainlit/playground/providers/anthropic.py#L32) example.

Reply:
Also can we pass the settings directly to the stream method? Does this work for all Langchain LLMs? Just curious

Reply:
This works so long as the `id` of the setting matches a parameter name in the chat model function. Langchain seems to use the same name convention for all of their extra parameters for chat models

- https://api.python.langchain.com/en/latest/chat_models/langchain_community.chat_models.anthropic.ChatAnthropic.html?highlight=chatanthropic#
- https://api.python.langchain.com/en/latest/chat_models/langchain_community.chat_models.ollama.ChatOllama.html?highlight=chatollama#langchain_community.chat_models.ollama.ChatOllama

However, this is not always the case, like in https://api.python.langchain.com/en/latest/chat_models/langchain_community.chat_models.openai.ChatOpenAI.html?highlight=chatopenai#langchain_community.chat_models.openai.ChatOpenAI.